### PR TITLE
IGNITE-13876 Updated documentation for 2.9.1 release

### PR DESCRIPTION
**Pull request by ymolochkov**

### CODE REVIEW

File: docs/_docs/monitoring-metrics/system-views.adoc [hunk 1]
```diff
@@ -676,3 +676,30 @@ This view exposes information about the distribution of cache group partitions a
 |STATE | string | Partition state. Possible states: MOVING - partition is being loaded from another node to this node; OWNING - this node is either a primary or backup owner; RENTING - this node is neither primary nor back up owner (is being currently evicted); EVICTED - partition has been evicted; LOST - partition state is invalid, the partition should not be used.
 |IS_PRIMARY | boolean  | Primary partition flag
 |===
+
+== BINARY_METADATA
+
+This view exposes information all available binary types.
```
Comment (Nikita-tech-writer) [inline docs/_docs/monitoring-metrics/system-views.adoc:682]:
```suggestion
This view exposes information about all available binary types.
```

**Review by Nikita-tech-writer — CHANGES_REQUESTED** [review]
Please see my suggesed change to your PR.

**Comment by ymolochkov** [comment]
> Please see my suggesed change to your PR.

Thanks, applied your suggestion

**Review by Nikita-tech-writer — APPROVED** [review]

**Review by Nikita-tech-writer — APPROVED** [review]

**Comment by dmagda** [comment]
@ymolochkov merged to the master! Please work with 2.9.1 release manage to cherry-pick the commit to the release branch. Once the vote is closed, ping me via email or Slack and I'll upload 2.9.1 docs to the website.

**Comment by ymolochkov** [comment]
@dmagda but what should we do with docs if they are in separate branch (2.9-docs)? Create another branch for the same purposes or merge docs to the release branch?

**Pull request merged**